### PR TITLE
Update application.md

### PR DESCRIPTION
**Pull request by harikt**
Excpetions were thrown as mentioned in https://github.com/Monofony/Skeleton/issues/94 .

### CODE REVIEW

File: docs/setup/application.md [hunk 1]
```diff
@@ -86,6 +86,16 @@ And execute the following commands:
 composer require monofony/api-pack "^0.9"
 ```
 
+If you happen to see exception as below, run the migration commands.
+
+> Class "App\Entity\RefreshToken" sub class of "Gesdinet\JWTRefreshTokenBundle\Entity\RefreshToken" is not a valid entity or mapped super class.
+
+```
```
Comment (loic425) [inline docs/setup/application.md]:
```suggestion
```bash
```

File: docs/setup/application.md [hunk 2]
```diff
@@ -86,6 +86,16 @@ And execute the following commands:
 composer require monofony/api-pack "^0.9"
 ```
 
+If you happen to see exception as below, run the migration commands.
```
Comment (loic425) [inline docs/setup/application.md]:
```suggestion
If you see an exception as shown below, run the migration commands.
```

**Comment by harikt** [comment]
Changes are committed

**Review by loic425 — APPROVED** [review]

**Comment by loic425** [comment]
thanks @harikt 🎉

**Pull request merged**